renovate.json

```diff
@@ -1,12 +1,17 @@
 {
   "extends": ["config:base"],
+  "prCreation": "not-pending",
   "packageRules": [
     {
       "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+      "matchCurrentVersion": "!/^0/",
```

Comment:
Is the thinking just that 0.whatever packages are probably a little unstable and any change needs checking out?

If we have packages that don't obey semver (Monaco, for example, doesn't seem to) do think we should tailor this further so we don't accidentally automerge them?

Reply:
Thinking about it, Monaco was a bad example since it would already be caught by this rule, but you get the gist.

Reply:
This change will no longer merge < 1 packages.